local-cli/generator-macos/templates/macos/Podfile

```diff
@@ -6,12 +6,7 @@ target 'HelloWorld-macOS' do
   use_native_modules!
   use_react_native!(
     :path => '../node_modules/react-native-macos',
-    
-    # To use Hermes, install the `hermes-engine-darwin` npm package, e.g.:
-    #   $ yarn add 'hermes-engine-darwin@~0.5.3'
-    #
-    # Then enable this option:
-    #   :hermes_enabled => true
+    # :hermes_enabled => true
```

Comment:
Should we sync to the iOS template with a comment that says what this does? See https://github.com/facebook/react-native/blob/0.66-stable/template/ios/Podfile

Reply:
I initially had this as a copy of the iOS template podifle with extra lines for macOS, but that didn't build. I sucessively removed changes till I got to this smallest pod file that does build. I can go back and add the comment again.